SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+## Supported Versions
+
+Information about supported versions and their security update status will be documented here as the project matures and establishes a formal release cadence.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| TBD     | :white_check_mark: |
+| < X.Y   | :x:                |
+
+## Reporting a Vulnerability
+
+The KubeOps team and community take security bugs seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+To report a security vulnerability, please use the [GitHub Security Advisory "Report a Vulnerability" tab](https://github.com/ewassef/dotnet-operator-sdk/security/advisories/new).
+
+Alternatively, you can report a security vulnerability by emailing the project maintainers at [INSERT PROJECT SECURITY EMAIL ADDRESS].
```

Comment:
also mail here?

Reply:
I'd say they should use the discussion or issue board for this. Instead of email